tripleblindmarket/private-kit
# SAF-685 Fix multiple clickable items share a screen location

### Description:

`<Button>` component is wrapped in `<MenuTrigger>` on the Choose Provider screen and both are clickable. As per SAF-685, Google Accessibility Scanner highlighted this as an issue. Disabling `<MenuTrigger>` prevents it from being clickable. 


#### Linked issues:

Fixes SAF-685 https://pathcheck.atlassian.net/browse/SAF-685

#### How to test:

Scan "Add Healthcare Authority" button on ChooseProvider view with Google Accessibility Scanner to see if error still shows up.
